File: Test_disp4.py
# Test Source
# リストオブジェクト anc --> ABC
list_obj = ["apple", "orange", "banana"]
for fruit_name in list_obj:

    upper_name = fruit_name.upper()
    print(upper_name)

# タプルオブジェクト anc --> ABC
tuple_obj = ("apple", "orange", "banana")
for fruit_name in tuple_obj:

    upper_name = fruit_name.upper()
    print(upper_name)

# 文字列オブジェクト hello --> HELLO
str_obj = "hello"
for letter in str_obj:

    upper_letter = letter.upper()
    print(upper_letter)

# 辞書オブジェクト English --> Japanese
dict_obj = {"dog": "犬", "cat": "猫", "parrot": "オウム"}
for english in dict_obj:

    japanese = dict_obj[english]
    print(english, "は", japanese, "です。")

File: test_disp4.php
<?php
// リストオブジェクト anc --> ABC
$list_obj = ["apple", "orange", "banana"];
foreach ($list_obj as $fruit_name) {
    $upper_name = strtoupper($fruit_name);
    echo $upper_name . PHP_EOL;
}

// タプルオブジェクト anc --> ABC
// PHPにはタプルはありませんが、配列で代用します
$tuple_obj = ["apple", "orange", "banana"];
foreach ($tuple_obj as $fruit_name) {
    $upper_name = strtoupper($fruit_name);
    echo $upper_name . PHP_EOL;
}

// 文字列オブジェクト hello --> HELLO
$str_obj = "hello";
for ($i = 0; $i < strlen($str_obj); $i++) {
    $upper_letter = strtoupper($str_obj[$i]);
    echo $upper_letter . PHP_EOL;
}

// 辞書オブジェクト English --> Japanese
$dict_obj = ["dog" => "犬", "cat" => "猫", "parrot" => "オウム"];
foreach ($dict_obj as $english => $japanese) {
    echo $english . " は " . $japanese . " です。" . PHP_EOL;
}
?>